detection-rules/link_download_disk_image_in_encrypted_zip.yml

```diff
@@ -1,10 +1,11 @@
-name: "Download link to disk image in encrypted zip"
+name: "Link to auto-downloaded disk image in encrypted zip"
 description: |
   A link in the body of the email downloads an encrypted zip that contains a disk image of the format IMG, ISO or VHD. This is a combination of file types used to deliver Qakbot.
 type: "rule"
 references:
   - "https://twitter.com/pr0xylife/status/1592502966409654272"
   - "https://delivr.to/payloads?id=ca00292e-d5a2-43f9-b638-6c0b01b73353"
+  - "https://www.trendmicro.com/en_us/research/22/j/black-basta-infiltrates-networks-via-qakbot-brute-ratel-and-coba.html"
 severity: "medium"

```

Comment:
one last gut-check, should this be "high"?

Reply:
could be, let's revisit later